src/iad.cs

```diff
@@ -172,7 +172,8 @@ interface ADInterstitialAdDelegate {
 		void WillLoad (ADInterstitialAd interstitialAd);
 	}
 
-	[Category, BaseType (typeof (MPMoviePlayerController))]
+	[Category (ignoreStaticWarnings: true)]
```

Comment:
@dalexsoto we should not consider anything in `XAMCORE_4_0` as final - but just our current best guess of a better profile. But having them in place will **force** us to consider it :)